test/integration.js_10.snap

```diff
@@ -337,12 +322,12 @@ exports[`c8 report generates report from existing temporary files 1`] = `
 ",--------------------------|---------|----------|---------|---------|--------------------------------
 File                      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s              
 --------------------------|---------|----------|---------|---------|--------------------------------
-All files                 |    73.6 |    59.03 |    62.5 |    73.6 |                                
+All files                 |   73.72 |    59.03 |    62.5 |   73.72 |                                
  bin                      |   78.84 |       60 |   66.66 |   78.84 |                                
   c8.js                   |   78.84 |       60 |   66.66 |   78.84 | 22,27-29,32-33,41-43,50-51     
- lib                      |   78.19 |    54.38 |      72 |   78.19 |                                
+ lib                      |   78.32 |    54.38 |      72 |   78.32 |                                
```

Comment:
It's weird to me that this coverage includes information from `c8` itself in Node 10, but that Node 12+ does not.

I checked the raw output from Node.js, and it's an actual discrepancy between the two outputs. We'll be dropping Node 10 soon, so perhaps this is okay.